practice-one/src/styles/sections/service.css

```diff
@@ -50,3 +44,27 @@
 .service-button {
   margin-top: 50px;
 }
+
+.service:nth-child(n)::before {
```

Comment:
nth-child(n) use for?

Reply:
The :nth-child(n) pseudo-class in CSS is used to select elements based on their position in a parent element. The n can be a number, keyword, or formula.

Reply:
But you don't use n here?

Reply:
I knew where is my mistake and fixed it, thanks for your contribution